<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLightControl</class>
 <widget class="QDockWidget" name="CLightControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QToolButton" name="light_mod_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="light_mod_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QSlider" name="light_brightness">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>22</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="light_mod_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="light_on">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="light_mod_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="lightCirclePanel" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="light_off">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
